packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/e2e/fixture.ts

```diff
@@ -0,0 +1,43 @@
+import {DocumentSuggestionObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {CaseAssistObject} from '../../../../../../playwright/page-object/caseAssistObject';
+
+const pageUrl = 's/quantic-document-suggestion';
+
+interface DocumentSuggestionOptions {
+  engineId: string;
+  searchEngineId: string;
+  maxDocuments: number;
+  fetchOnInit: boolean;
+  withoutQuickview: boolean;
+  numberOfAutoOpenedDocuments: number;
+}
+
+type QuanticDocumentSuggestionE2EFixtures = {
+  documentSuggestion: DocumentSuggestionObject;
+  search: SearchObject;
+  caseAssist: CaseAssistObject;
+  options: Partial<DocumentSuggestionOptions>;
+};
+
+type QuanticDocumentSuggestionE2ESearchFixtures =
```

Comment:
do we need this extra type or could we just shove the urlHash in the same type as the rest?

Reply:
Ouai dans le fond lol